src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -691,19 +691,27 @@
      *
      * <p>For example:</p>
      * <pre>
-     * Math.random()
+     * String n = Integer.parseInt("123");
```

Comment:
The example should be as short as possible. Please remove variable assignment.

Reply:
Can I take Math.random() only.

Reply:
Is this correct?
```
**Test.java**
public class Test {
	void test() {
		Integer.parseInt("123");
	}
}

# Expected update
 `--METHOD_CALL -> (
     |--DOT -> .
     |   |--IDENT -> Integer
     |   `--IDENT -> parseInt
     |--ELIST -> ELIST
     |   `--EXPR -> EXPR
     |       `--STRING_LITERAL -> "123"
     `--RPAREN -> )
```

Reply:
yes, this looks very clear.